editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Search/SearchHeader.tsx

```diff
@@ -23,6 +24,14 @@ export const SearchHeader: Components<Data, Context>["Header"] = ({
     }
   }, [formik.values.pattern, lastPattern, setIsSearching]);
 
+  // Use "data.title" as proxy for all facets
+  const isDataOnly = () => !formik.values.facets.includes("data.title");
+
+  const toggleDataOnly = () =>
+    isDataOnly()
+      ? formik.setFieldValue("facets", ALL_FACETS)
+      : formik.setFieldValue("facets", DATA_FACETS);
```

Comment:
The logic here feels backwards to me based on naming?

Reply:
Thanks, updated naming 👍